src/frontend/app/shared/components/cards/service-summary-card/service-summary-card.component.html

```diff
@@ -43,6 +43,12 @@
       </a>
     </app-meta-card-value>
   </app-meta-card-item>
+  <app-meta-card-item>
```

Comment:
We see some services without a provider, so something like this would work for all
```
<app-meta-card-item *ngIf="servicesService.getServiceProviderName() | async as providerName">
    <app-meta-card-key>Provider</app-meta-card-key>
    <app-meta-card-value>{{providerName}}</app-meta-card-value>
  </app-meta-card-item>
```